src/main/kotlin/racingcar/view/ErrorView.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar.view
+
+object ErrorView {
+    fun errorMessage(message: String) {
+        println(message)
+    }
```

Comment:
이렇게 단일표현식을 활용하시면 보다 간결하게도 표현할 수 있습니다.
```suggestion
    fun errorMessage(message: String) = println(message)
```

Reply:
코드를 조금 더 간결하게 만들 수 있겠군요~!! 알려주셔서 감사해요